Sources/App/Models/Repository.swift

```diff
@@ -114,6 +114,9 @@ final class Repository: Model, Content {
     @Field(key: "summary")
     var summary: String?
 
+    @Field(key: "last_activity_at")
+    var lastActivityAt: Date?
+
```

Comment:
Nit: I've started a (feeble) attempt at ordering the fields alphabetically a while back when we got more than a handful of fields. It's hardly consistent because I started late and some initialisers would be a pain to have their arguments re-ordered. This one we could bump up to after `keywords` though.